core/src/main/java/tc/oc/pgm/observers/tools/GamemodeTool.java

```diff
@@ -20,6 +20,12 @@
 
 public class GamemodeTool implements InventoryMenuItem {
 
+  final InventoryMenu in;
+
+  public GamemodeTool(InventoryMenu in) {
+    this.in = in;
```

Comment:
```suggestion
  final InventoryMenu menu;

  public GamemodeTool(InventoryMenu menu) {
    this.menu = menu;
```

Reply:
Wops this is just code I forgot to remove